### DISCUSSION TITLE
2022-08-22-PowerShell-Citrix-Hypervisor-Count-Function/

### DISCUSSION THREAD

**Discussion by giscus** [alainassaf/alainassaf.github.io#3] [Site Comments]
# 2022-08-22-PowerShell-Citrix-Hypervisor-Count-Function/

Contents Contents Scenario Citrix Hypervisor and PowerShell? Conculsion Learning More Value for Value Scenario I’ve been hesitant to dive into Citrix Hypervisor PowerShell cmdlets, but there’s no rational reason to not do it. Citrix continues to make great strides in expanding and updating PowerShell for its hypervisor, PVS, and XenDesktop....

https://alainassaf.com/2022-08-22-PowerShell-Citrix-Hypervisor-Count-Function/?utm_source=blog&utm_medium=blog&utm_content=recent

**Comment by alainassaf** [comment]
Testing giscus comment.

**Comment by ghost** [comment]
testing comment

**Comment by OsmondOlive** [comment]
testing
